### ISSUE TITLE
Add a notice for the trusted hosts allowed as avatar urls.

### ISSUE THREAD

**Issue by kz-n** [xornet-cloud/Client#45]
I noticed while trying to update my avatar url, I kept getting a 400 status code, and that was because I copied the image image link from my GitHub profile.
I found out about this by looking at the code for `validator.ts`, perhaps knowing this could save someone in the future a search in the code.
![immagine](https://user-images.githubusercontent.com/64893077/154360692-94aada0e-e4db-4131-b69e-38422e535051.png)
Something like this

**Comment by Geoxor** [comment]
yeah i was too lazy to add something like this because i was planning on rewriting the profiles since they look shit, but no one wants to touch them besides me so i guess ill work on that next 💀💀💀
